kubeflow/trainer/rhai/traininghub.py

```
@@ -481,6 +522,39 @@ def _transform_sft(self, metrics):
+            epoch = metrics.get("epoch", 0)
+            max_steps = metrics.get("max_steps", 0)
+
+            # Calculate progress based on steps
+            percent = min(100, step / max_steps * 100) if max_steps > 0 else 0
+
+            loss_val = metrics.get("loss")
+            lr_val = metrics.get("learning_rate")
+            grad_norm_val = metrics.get("grad_norm")
+
+            return {
+                "progressPercentage": int(round(percent)),
```

Ruff (0.15.6)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)